tests/delete_test.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+"""Test Storage bucket classes."""
+
+import pytest
+from google.api_core.exceptions import Conflict
+from google.cloud.storage import Bucket
+from returns.pipeline import is_successful
+
+from storage_bucket.delete import DeleteBucket, delete_bucket
+
+
```

Comment:
maybe make fixture that takes a 'bucket_name' param to create a bucket first and then delete it?